spec/requests/audio_recordings/downloader_spec.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+describe '/audio_recordings/downloader' do
+  require 'helpers/shared_test_helpers'
+  extend WebServerHelper::ExampleGroup
+
+  include_context 'shared_test_helpers'
+  create_audio_recordings_hierarchy
+
+  def prepare_audio_file(audio_recording)
+    link_original_audio(
+      target: Fixtures.audio_file_mono,
+
+      uuid: audio_recording.uuid,
+      datetime_with_offset: audio_recording.recorded_date,
+      original_format: 'mp3'
+    )
+  end
+
+  describe 'the script is templated when downloaded' do
+    it 'contains the logged in user\'s name' do
+      get '/audio_recordings/downloader', headers: api_request_headers(reader_token)
+      expect_success
+
+      expect(response.body).to include('    $user_name = "reader"')
```

Comment:
Why all of the spaces? This may fail if you decide to change the script later.

Reply:
Intentionally brittle. I wanted to make sure that exact line with indentation was templated